docs/interface.rst

```diff
@@ -131,6 +129,69 @@ The response body follows the form of this example::
   {}
 
 
+``GET /v1/magic-folder/<folder-name>/participants``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+List all participants in a certain magic-folder.
+
+The response is **OK** with an ``application/json`` **Content-Type**::
+
+    {
+        "participant name": {
+            "personal_dmd": "URI:DIR2-RO:..."
+        }
+    }
+
+There will be one entry per participant.
+``personal_dmd`` is a Tahoe read-only directory capability-string.
+
+
+``POST /v1/magic-folder/<folder-name>/participants``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add a new participant to a certain magic-folder.
+Accepts a JSON body listing the details of the participant to add::
+
+    {
+        "author": {
+            "name": "arbitrary string"
+        },
+        "personal_dmd": "URI:DIR2:..."
+    }
+
+The response is delayed until the participant is correctly added to the Collective DMD.
+``personal_dmd`` is a Tahoe writable directory capability-string.
```

Comment:
I think that only the read-only capability for the personal DMD is required here.  "writeable directory capability-string" is perhaps ambiguous.  A careless reading might suggest it is a "write capability for a mutable directory" (and the example above seems to be such a thing).  I think "read-only mutable directory capability string" is what's really meant here - ie `"URI:DIR2-RO:..."`.  The implementation (`add_participant`) seems to really want `dmd.is_readonly()` to be `True` which seems in line with this.

Does that seem right to you?